src/encoded/schemas/death_circumstances.json

```diff
@@ -181,53 +134,52 @@
             "minimum": 1,
             "maximum": 4
         },
-        "icd_10_category": {
-            "title": "Icd 10 Category",
-            "description": "Category of death based on ICD-10 coding",
-            "type": "string"
-        },
-        "icd_10_cause": {
-            "title": "Icd 10 Cause",
-            "description": "Specific cause of death based on ICD-10 coding ",
-            "type": "string"
-        },
-        "icd_10_classification": {
-            "title": "Icd 10 Classification",
-            "description": "Classification of death based on ICD-10 coding ",
-            "type": "string"
-        },
-        "icd_10_code": {
-            "title": "Icd 10 Code",
-            "description": "ICD-10 Code for cause of death",
-            "type": "string"
-        },
-        "last_seen_alive_datetime": {
-            "title": "Last Seen Alive Datetime",
-            "description": "Date and time when the donor was last known to be alive",
+        "place_of_death": {
+            "title": "Place Of Death",
+            "description": "Place of death of the donor",
             "type": "string",
-            "format": "date-time"
+            "enum": [
+                "Home",
+                "Hospital",
+                "Other"
+            ]
         },
-        "manner_of_death": {
-            "title": "Manner Of Death",
-            "description": "Manner of death of the donor",
+        "region_of_death": {
```

Comment:
This assumes US based donors, which may be what is expected but may want to add explicitly in description and if any possibility that not US then add an 'Other' option.